lib/web/apiserver.go

```diff
@@ -3731,12 +3752,102 @@ func (h *Handler) WithClusterAuth(fn ClusterHandler) httprouter.Handle {
 	})
 }
 
+func (h *Handler) writeErrToWS(ws *websocket.Conn, err error) {
```

Comment:
```suggestion
func (h *Handler) writeErrToWebsocket(ws *websocket.Conn, err error) {
```

Let's spell out websocket - it's not that much to type and `WS` could also be taken to mean `WebSession`.